lib/chai/utils/overwriteProperty.js

```diff
@@ -4,7 +4,9 @@
  * MIT Licensed
  */
 
+var chai = require('../../chai');
```

Comment:
Does this create a circular dependency? Have we done this before? I feel like this might create issues